Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/UnitData.xml

```diff
@@ -2253,8 +2256,30 @@
     </CUnit>
     <CUnit id="AP_GhostResourceEfficiency" parent="AP_Ghost">
         <AbilArray index="10" removed="1"/>
-        <BehaviorArray index="2" removed="1"/>
+        <BehaviorArray index="2" Link="AP_GhostPermanentCloak"/>
+        <BehaviorArray index="3" removed="1"/>
         <CargoSize value="1"/>
+        <CardLayouts index="0">
+            <LayoutButtons index="10">
+                <Face value="CloakOff"/>
+                <Type value="AbilCmd"/>
+                <AbilCmd value="AP_GhostCloak,Off"/>
+                <Requirements value=""/>
+                <Row value="2"/>
+                <Column value="3"/>
+            </LayoutButtons>
+            <LayoutButtons index="11" Face="AP_EMP" Type="AbilCmd" AbilCmd="AP_EMP,Execute" Row="2" Column="1"/>
+            <LayoutButtons index="12" Face="AP_WeaponsFree" Type="AbilCmd" AbilCmd="AP_GhostWeaponsFree,Execute" Row="1" Column="3"/>
+            <LayoutButtons index="13" Face="AP_GhostSCBWLockdown" Type="AbilCmd" AbilCmd="AP_GhostSCBWLockdown,Execute" Row="1" Column="1"/>
+            <LayoutButtons index="14">
+                <Face value="AP_CriusSuit"/>
+                <Type value="Passive"/>
+                <AbilCmd value=""/>
+                <Requirements value="AP_PermanentCloakGhostResearched"/>
+                <Row value="2"/>
+                <Column value="2"/>
+            </LayoutButtons>
+        </CardLayouts>

```

Comment:
There shouldn't be any button override. The Crius Suit button belongs to base `AP_Ghost`